src/main/groovy/io/gatling/gradle/GatlingPluginExtension.groovy

```diff
@@ -327,7 +325,8 @@ class GatlingPluginExtension {
         gatlingVersion = toolVersion
     }
 
-    Closure simulations = DEFAULT_SIMULATIONS
+    List<String> includes = List.of()
+    List<String> excludes = List.of()
```

Comment:
immutables?
Cannot do a `+=` ?

Reply:
We'll see if someone needs this. I can't see a use case of this.